app/controllers/applications_controller.rb

```diff
@@ -0,0 +1,44 @@
+class ApplicationsController < ApplicationController
+  def show
+    @application = Application.find(params[:id])
+    if params[:search_app].present?
+      @pets = Pet.search(params[:search_app])
+    end
+
+    if params[:pet_id].present?
+      @application.add_pet(params[:pet_id])  
+    end
+  end
+
+  def new
+  end
+
+  def create
+    admin = Admin.create
```

Comment:
`admin` in our path is actually not a model, but just an organizational tool. We can call it a namespace. It shouldn't actually map to a model or database table. So you can keep applications as a standalone object rather than something that belongs to an admin!